packages/flutter/test/widgets/platform_menu_bar_test.dart

```diff
@@ -53,6 +53,7 @@ void main() {
             child: PlatformMenuBar(
               menus: createTestMenus(
                 onActivate: onActivate,
+                onActivateIntent: (String item) => TestPlatformMenuBarIntent(item),
```

Comment:
If you change the argument type in `createTestMenus`, this can just be:
```suggestion
                onActivateIntent: TestPlatformMenuBarIntent(subMenu1[0]),
```